### PR TITLE
[B2BP-589] - IFrame Component

### PR DESCRIPTION
**Pull request by salvatorediocaro**
#### List of Changes

Created the IFrame.tsx and IFrame.types.ts files

#### Motivation and Context

Import the content of an external page via IFrame

#### How Has This Been Tested?

Execute on locally B2B-Portals

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/components/IFrame.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+"use client";
+import { IFrame as IFrameRC } from "@react-components/components";
+import { IFrameSectionProps } from "@/lib/fetch/types/PageSection";
+
+const IFrame = (props: IFrameSectionProps) => (
```
Comment (simosalsi) [inline apps/nextjs-website/src/components/IFrame.tsx:5]:
```suggestion
const IFrameSection = (props: IFrameSectionProps) => (
```
For consistency

File: apps/nextjs-website/src/components/IFrame.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  <IFrameRC {...props} />
+);
+
+export default IFrame;
```
Comment (simosalsi) [inline apps/nextjs-website/src/components/IFrame.tsx:9]:
```suggestion
export default IFrameSection;
```
For consistency

File: apps/nextjs-website/react-components/components/IFrame/iframe-resizer.d.ts
Comment (simosalsi) [inline apps/nextjs-website/react-components/components/IFrame/iframe-resizer.d.ts:1]: Why is this needed?

**Review by simosalsi — CHANGES_REQUESTED** [review]

**Review by simosalsi — DISMISSED** [review]

**Review by maurodandrea — DISMISSED** [review]

**Review by simosalsi — APPROVED** [review]
Approving to demo the iFrame

Commented code inside iFrame will be reinstated when client side issues are resolved.

**Review by simosalsi — APPROVED** [review]
Approving to demo the iFrame

Commented code inside iFrame will be reinstated when client side issues are resolved.

**Pull request merged**